src/OSS/Http/RequestCore.php

```diff
@@ -566,10 +600,18 @@ public function streaming_read_callback($curl_handle, $file_handle, $length)
      */
     public function streaming_write_callback($curl_handle, $data)
     {
+        $code = curl_getinfo($curl_handle, CURLINFO_HTTP_CODE);
+        
+        if (intval($code) / 100 != 2)
+        {
+            $this->response_error = $data;
```

Comment:
- response_error  -> response_error_body
-  if  $data != null  $this->response_error .= $data;